app/models/manageiq/providers/vmware/infra_manager/provision_workflow.rb

```diff
@@ -131,7 +131,7 @@ def available_vlans(options = {})
 
     unless @vlan_options[:dvs] == false
       rails_logger('allowed_dvs', 0)
-      vlans_dvs = allowed_dvs(@vlan_options, hosts)
+      vlans_dvs = allowed_dvs
```

Comment:
@lfu We should retaining the `hosts` loaded from the `super` call to `available_vlans` otherwise we will end up reloading and running rbac on them again.  I suggest changing the new method `available_vlans` to something like `available_vlans_and_hosts` and pass the `hosts` returned from that call to `allowed_dvs`.